Multi checkbox (#2289)
kyma-project/busola: docs/extensibility/form-section.md

<img src="./assets/form-widgets/Resource.png" alt="Example of a Resource widget" style="border: 1px solid #D2D5D9">

### MultiCheckbox

> MultiCheckbox widget render checkboxes that will be put saved into one path as array of strings.

Reviewer: ```suggestion
MultiCheckbox widgets render checkboxes that are saved into one path as an array of strings.
```

#### Widget-specific parameters

- **options[]** - _[required]_ an array of objects to generate the checkboxes.

>   - **key** - _[required]_ a value used to generate checkboxes. Will be set if checkbox is selected. This `key` together with `path` defined for MultiCheckbox (`{path}.{key}`) can be used for **translation**.

Reviewer: ```suggestion
  - **key** - _[required]_ a value used to generate checkboxes. It is set if a checkbox is selected. This `key`, together with the `path` defined for MultiCheckbox (`{path}.{key}`), can be used for **translation**.
```
